### PR TITLE
Documentation/initial partial edits

### PR DESCRIPTION
**Pull request by nikomancy**
### 🎯 Goal

Refine the content in the setup section

### 🛠 Implementation details

- Applied edits for concision.
- Applied edits to reduce use of passive voice.
- Applied edits maintain consistent terminology.
- Added content around developer tokens.
- Added links to improve navigability.

### 🧪 Testing
- It's content

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] ~~Changelog is updated with client-facing changes~~
- [ ] ~~New code is covered by unit tests~~
- [ ] ~~Comparison screenshots added for visual changes~~
- [x] Affected documentation updated (CMS, cookbooks, tutorial)
- [x] Reviewers added

### CODE REVIEW

File: documentation/docs/client/setup/initializing_sdk.md [hunk 1]
```diff
@@ -4,26 +4,32 @@ title: Initializing SDK
...
+         val chatClient = ChatClient.Builder("apiKey", context).build()
+
          // Static reference to initialised client
          val staticClientRef = ChatClient.instance()
      }
  }
  ```
+> You can access your apiKey in the [Dashboard](https://getstream.io/dashboard)
+
+If you create the `ChatClient` instance following the pattern in the previous example, you will be able to access the that instance from any part of your application.
```
Comment (zsmb13) [inline documentation/docs/client/setup/initializing_sdk.md:24]:
```suggestion
If you create the `ChatClient` instance following the pattern in the previous example, you will be able to access that instance from any part of your application. 
```

File: documentation/docs/client/setup/connecting_user.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Connecting the User
 sidebar_position: 3
 ---
 
-The next step is connecting the user. A valid `StreamChat` token is all you need to properly set your current app user as the current user of `ChatClient`. This token can't be created locally and it must be provided by your backend.
+The next step is connecting the user. A valid `StreamChat` token is all you need to properly set your current app user as the current user of `ChatClient`. Because you must use your `API_SECRET` to create this token, it is unsafe to generate this token outside of a secure server. This token can't be created locally and it must be provided by your backend.
```
Comment (zsmb13) [inline documentation/docs/client/setup/connecting_user.md:7]: The last two sentences seem like duplicates here

File: documentation/docs/client/setup/initializing_sdk.md [hunk 3]
```diff
@@ -4,26 +4,32 @@ title: Initializing SDK
 sidebar_position: 2
 ---
 
- As a first step, you need to initialize `ChatClient`, which is the main entry point for all operations in the library. You should only create the client once and re-use it across your application. Typically `ChatClient` is initialized in `Application` class:
+Integrating with the you need to initialize the `ChatClient`. The `ChatClient` is the main entry point for all operations in the library. Create a single `ChatClient` and re-use it across your application.
```
Comment (zsmb13) [inline documentation/docs/client/setup/initializing_sdk.md:7]: > Integrating with the you

Not sure what happened here

File: documentation/docs/client/setup/initializing_sdk.md [hunk 4]
```diff
@@ -4,26 +4,32 @@ title: Initializing SDK
...
+
+A best practice is to ininitialize `ChatClient` in the `Application` class and create a static reference to the client:
```
Comment (zsmb13) [inline documentation/docs/client/setup/initializing_sdk.md:9]:
```suggestion
A best practice is to initialize `ChatClient` in the `Application` class and create a static reference to the client:
```

File: documentation/docs/client/setup/logging.md [hunk 5]
```diff
@@ -30,16 +38,16 @@ val client = ChatClient.Builder("apiKey", context)
      .build()
 ```
 
-## Finding logs
+## Filtering logs
 
-All SDK log tags have the `Chat:` prefix, so you can filter for that those in the logs:
+All SDK log tags `Chat:` as a prefix that you can use when filtering logs.
```
Comment (zsmb13) [inline documentation/docs/client/setup/logging.md:43]:
```suggestion
All SDK log tags have `Chat:` as a prefix that you can use when filtering logs.
```

File: documentation/docs/client/setup/logging.md [hunk 6]
```diff
@@ -30,16 +38,16 @@ val client = ChatClient.Builder("apiKey", context)
...
 adb logcat com.your.package | grep "Chat:"
 ```
 
 Here's a set of useful tags for debugging network communication:
 
-- `Chat:Http`
-- `Chat:Events`
-- `Chat:SocketService`
+- `Chat:Http` to see HTTP requests made from the `ChatClient` and the responses returned by Stream.
+- `Chat:Events` to see a list of [events](https://getstream.github.io/stream-chat-android/stream-chat-android-client/stream-chat-android-client/io.getstream.chat.android.client.events/-chat-event/index.html) that the `ChatClient` emits.
```
Comment (zsmb13) [inline documentation/docs/client/setup/logging.md:52]: We'll want to update this link when things are moved to Docusaurus, could be worth adding a TODO for somehow (or omitting for now)

**Review by zsmb13 — COMMENTED** [review]
Couple thoughts / suggestions

File: documentation/docs/client/setup/logging.md (hunk 6 above)
Comment (nikomancy) [reply documentation/docs/client/setup/logging.md:52]: My plan is to keep a list of these old links and use a regex to replace them with analogous pages when we migrate. Do you know if the Dokka docs will have the same paths after the base url for the github pages site? (IE: of `https://getstream.github.io/stream-chat-android/stream-chat-android-client/stream-chat-android-client/io.getstream.chat.android.client.events/-chat-event/index.html` will everything after `https://getstream.github.io/stream-chat-android` remain the same or are we changing the paths for the API reference?)

File: documentation/docs/client/setup/connecting_user.md (hunk 2 above)
Comment (nikomancy) [reply documentation/docs/client/setup/connecting_user.md]: Let me remove that sentence. Good call.

File: documentation/docs/client/setup/initializing_sdk.md (hunk 3 above)
Comment (nikomancy) [reply documentation/docs/client/setup/initializing_sdk.md:7]: Typos, typos happened 😞 . Let me fix that.

**Review by nikomancy — COMMENTED** [review]
That should resolve those mistakes.

File: documentation/docs/client/setup/logging.md (hunk 6 above)
Comment (JcMinarro) [reply documentation/docs/client/setup/logging.md:52]: I have already create a task for it

Comment (JcMinarro) [reply documentation/docs/client/setup/logging.md:52]: Dokka Reference will live into our Docusaurus project, and it should have similar url from the root of this project

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**